dali/base/dadfs.cpp

```diff
@@ -4683,7 +4685,7 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
 
     CDistributedFilePartArray partscache;
     FileClusterInfoArray clusterscache;
-
+    bool subfileRestricted;
```

Comment:
should initialize to false here.
It may not be possible to reference before initialized at the moment, but safer, more readable and inconsequential cost.

Generally better to initialize all members at member declaration time.